src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -50,7 +54,7 @@ public class ResourceManagerStringLocalizer : IStringLocalizer
         /// </summary>
         public ResourceManagerStringLocalizer(
             ResourceManager resourceManager,
-            AssemblyWrapper resourceAssemblyWrapper,
+            IResourceStreamManager resourceStreamManager,
```

Comment:
Unfortunately the opposite could also be true: Since `ResourceManagerStringLocalizer` was public, we might need to consider every constructor parameter type public. Either way the `.Internal` stuff was a bit borked here.